package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "manh-react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "1.4.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-router-dom": "6.3.0",
+    "react-scripts": "5.0.1",
+    "sass": "1.49.11"
+  },
+  "scripts": {
+    "start": "react-scripts -r @cypress/instrument-cra start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --transformIgnorePatterns",
+    "eject": "react-scripts eject",
+    "test:coverage": "rimraf coverage && react-scripts test --coverage --watchAll=false && npm run cypress:run && node ./scripts/coverage-merge.js && nyc report",
+    "test:merge-coverage": "node ./scripts/coverage-merge.js",
+    "lint": "eslint ./ --ext .js,.ts,.tsx",
+    "lint:fix": "eslint ./ --ext .js,.ts,.tsx --fix",
+    "stylelint": "stylelint '**/*.scss'",
+    "stylelint:fix": "stylelint '**/*.scss' --fix",
+    "codebase:lint": "npm run lint && npm run stylelint",
+    "codebase:fix": "npm run lint:fix && npm run stylelint:fix",
+    "cypress": "start-server-and-test start 3000 cypress:run",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "7.21.0-placeholder-for-preset-env.2",
+    "@cypress/browserify-preprocessor": "3.0.2",
+    "@cypress/code-coverage": "3.9.12",
+    "@cypress/instrument-cra": "1.4.0",
+    "@nimblehq/eslint-config-nimble-react": "1.2.0",
+    "@nimblehq/eslint-config-nimble-testing": "1.1.0",
+    "@nimblehq/stylelint-config-nimble": "1.0.2",
+    "@testing-library/cypress": "9.0.0",
+    "@testing-library/jest-dom": "5.16.4",
+    "@testing-library/react": "14.0.0",
+    "@testing-library/user-event": "14.1.1",
+    "@types/jest": "29.5.2",
+    "@types/node": "20.2.5",
+    "@types/react": "18.2.9",
+    "@types/react-dom": "18.2.4",
+    "cypress": "12.14.0",
+    "cypress-react-selector": "3.0.0",
+    "danger": "11.2.6",
+    "danger-plugin-istanbul-coverage": "1.6.2",
+    "eslint": "8.12.0",
+    "postcss": "8.4.24",
+    "postcss-import": "15.1.0",
+    "prettier": "2.6.0",
+    "start-server-and-test": "2.0.0",
+    "stylelint": "15.7.0",
+    "tailwindcss": "3.3.2",
+    "typescript": "4.6.2"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx,ts,tsx}",
+      "!src/**/*.d.ts"
+    ],
+    "coverageReporters": [
+      "json"
+    ]
+  },
```

Comment:
`jest` recommends extracting the configurations into a separate file, it also keeps the `package.json` a bit cleaner

<img width="1007" alt="image" src="https://github.com/manh-t/react-survey/assets/39977805/65e05406-aaf0-4820-a011-a7b34062a352">

could we do that here

```js
// jest.config.js
/** @type {import('jest').Config} */
const config = {
  collectCoverageFrom: [
      "src/**/*.{js,jsx,ts,tsx}",
      "!src/**/*.d.ts"
    ],
    coverageReporters: [
      "json"
    ]
};

export default config;
```

Reply:
Thanks, I create `jest.config.ts` in [90dfded](https://github.com/manh-t/react-survey/pull/39/commits/90dfdedb959020ae7eaf841bb008a4f8a16dde6b) 🎉